packages/server/tests/utils.ts

```diff
@@ -175,8 +177,8 @@ export default class TestUtils {
             .setOperator(AccountId.fromString(id), opPrivateKey);
     };
 
-    createEthCompatibleAccount = async (client: Client) => {
-        const privateKey = PrivateKey.generateECDSA();
+    createEthCompatibleAccount = async (client: Client, privateKeyHex: string) => {
```

Comment:
Is there a reason why we are not generating the ECDSA but rather hardcoding them?

Reply:
Because if they are randomly generated the transaction hash in the result will be different every time and that makes it harder to assert.